asmcomp/arm/selection.ml

```diff
@@ -179,8 +179,9 @@ method select_shift_arith op dbg arithop arithrevop args =
       | op_args -> op_args
       end
 
-method private iextcall (func, alloc) =
-  Iextcall { func; alloc; label_after = Cmm.new_label (); }
+method private iextcall func ty_res ty_args =
```

Comment:
I found it confusing that `ty_res` and `ty_args` are in this order; could we reverse them?  I think this would make the code that follows easier to read.

Reply:
It's like a C function type :-)  I'm not very keen on it but I think it does the job, so I left the code as is.